helm/ffc-demo-claim-service/development-values.yaml

```diff
@@ -1,5 +1,4 @@
 container:
   command: ["/bin/sh","-c"]
-  args: ["scripts/wait-for/wait-for $(POSTGRES_HOST):5432 -- npm --no-update-notifier run migrate && node index"]
-  messageQueueHost: ffc-demo-claim-artemis-activemq-artemis
+  args: ["node index"]
```

Comment:
Is there any value in keeping this file if we're going to drop support for local deployment?

There's a separate ticket to do that in all repos but no objection to this being removed now.

Reply:
Fair point, I think I'll leave it for the other ticket, thanks.